diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+_/Users/grace/wdi/try_
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+_It's a repository for storing files for sharing._
```

Comment:
Yes, but more specifically: Github is a user interface (UI) for accessing git's version control capabilities for our project repositories.